JobConfig/digitize/prolog.fcl

```diff
@@ -189,5 +211,8 @@ Digitize.Outputs : {
 # give paths specific numbers outside trigger path range
 Digitize.trigger_paths : ["0:DigitizePath", "1:TriggerablePath", "2:TriggerableCHPath" ]
 Digitize.EndPath : [ @sequence::Digitize.EndSequence, TriggeredOutput, TriggerableOutput ]
-
+# override the trigger digitization sequence
+TrigRecoSequences.artFragmentsGen : @local::Digitize.DigitizeSequence
```

Comment:
This additionally adds the CRV digi sequence, which I assume is fine but I'm just noting this change.

Reply:
CRV needs to be there, as otherwise the MC matching fails. It's a negligible cost, as it has to be called anyways on any event which triggers.